src/d3a/models/area/event_dispatcher.py

```diff
@@ -151,9 +151,6 @@ def create_area_agents(self, is_spot_market, market):
                 owner=self.area,
                 higher_market=self.area.parent._markets.markets[market.time_slot],
                 lower_market=market,
-                transfer_fee_pct=0
-                if ConstSettings.IAASettings.AlternativePricing.PRICING_SCHEME != 0
```

Comment:
I dont see this logic in the area (or market). Can you please double check with the science team whether it is still wanted, that the transfer_fee_pct is 0 for alternative Pricing schemes? Thanks!

Reply:
Thanks for pointing it, moving this logic to Area class.